.github/workflows/ci.yml

```diff
@@ -383,108 +383,10 @@ jobs:
-          PublisherUrl: https://dotenvx.com
-          Author: motdotla
-          PackageName: dotenvx
-          PackageUrl: https://github.com/dotenvx/dotenvx
-          License: BSD-3-Clause
-          ShortDescription: a better dotenv–from the creator of dotenv
-          Description: a better dotenv–from the creator of dotenv
-          Tags: ["env", "dotenv", "environment", "variables", "dotenvx", "cli", "encryption"]
-          Moniker: dotenvx
-          ManifestType: defaultLocale
-          ManifestVersion: 1.6.0
-          EOT
-      - name: calculate sha256 for x64 installer
-        run: |
-          INSTALLER_URL="https://github.com/dotenvx/dotenvx/releases/download/${{ env.VERSION }}/dotenvx-${{ env.CLEAN_VERSION }}-windows-x86_64.zip"
-          curl -L $INSTALLER_URL -o dotenvx-${{ env.CLEAN_VERSION }}-windows-x86_64.zip
-          SHA_X86=$(sha256sum dotenvx-${{ env.CLEAN_VERSION }}-windows-x86_64.zip | awk '{ print $1 }')
-          echo "SHA_X86=$SHA_X86" >> $GITHUB_ENV
-      - name: create dotenvx.dotenvx.installer.yaml
-        run: |
-          cd winget-pkgs
-          mkdir -p manifests/d/dotenvx/dotenvx/${{ env.CLEAN_VERSION }}
-          CURRENT_DATE=$(date +"%Y-%m-%d")
-          cat <<EOT > manifests/d/dotenvx/dotenvx/${{ env.CLEAN_VERSION }}/dotenvx.dotenvx.installer.yaml
-          PackageIdentifier: dotenvx.dotenvx
-          PackageVersion: ${{ env.CLEAN_VERSION }}
-          InstallerLocale: en-US
-          InstallerType: zip
-          ReleaseDate: "$CURRENT_DATE"
-          Installers:
-            - Architecture: x64
-              NestedInstallerType: portable
-              NestedInstallerFiles:
-                - RelativeFilePath: dotenvx.exe
-                  PortableCommandAlias: dotenvx
-              InstallerUrl: https://github.com/dotenvx/dotenvx/releases/download/${{ env.VERSION }}/dotenvx-${{ env.CLEAN_VERSION }}-windows-x86_64.zip
-              InstallerSha256: ${{ env.SHA_X86 }}
-              UpgradeBehavior: uninstallPrevious
-          ManifestType: installer
-          ManifestVersion: 1.6.0
-          EOT
-      - name: commit and push
-        run: |
-          cd winget-pkgs
-          git add .
-          git commit -m "New version: dotenvx.dotenvx ${{ env.CLEAN_VERSION }}" --allow-empty
-          git branch
-          git push origin dotenvx-dotenvx-${{ env.CLEAN_VERSION }}
-      - name: create pull request
-        env:
-          GITHUB_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
-        run: |
-          cd winget-pkgs
-          gh pr create --repo microsoft/winget-pkgs \
-            --base master \
-            --head dotenvx:dotenvx-dotenvx-${{ env.CLEAN_VERSION }} \
-            --title "New version: dotenvx.dotenvx ${{ env.CLEAN_VERSION }}" \
-            --body "New version: dotenvx.dotenvx ${{ env.CLEAN_VERSION }}"
-
+      - name: Submit using wingetcreate
+        run: | 
+          iwr https://aka.ms/wingetcreate/latest -OutFile wingetcreate.exe
```

Comment:
thanks for this PR! looks potentially very helpful (removing a lot of boilerplate)

but can you point me to docs on this url? i'm hesitant to trust it initially as it downloads an executable remotely.

Reply:
It's what PowerToys uses in their release - https://github.com/microsoft/PowerToys/blob/main/.github/workflows/package-submissions.yml

Reply:
@denelon can potentially point to docs directly

Reply:
that's pretty trustworthy :). thanks. i'll try and test this out shortly.

Reply:
Let me know if you have any concerns :) I'm happy to help.

Reply:
We're hoping to get WinGet in a good state on the Windows images so you get the benefit of the package manager with hash validation and all the other goodies. Initially, it's likely to have an outdated version that won't work without some fiddling.

Reply:
ok, i think it worked: https://github.com/dotenvx/dotenvx/actions/runs/12439823801/job/34734350769

where do i go to view it now that it is not a PR request?

Reply:
nevermind. i see it here: https://github.com/microsoft/winget-pkgs/pull/200439

thanks for this guys! anything else you need from me let me know.

Reply:
Looks great!